src/database/postgres.ts

```diff
@@ -444,6 +444,12 @@ export class DatabaseWrapperPostgres {
 		});
 	}
 
+	async getConfirmed(tournamentId: string): Promise<DatabasePlayer[]> {
+		// is this how to use select properly? it seems off that it still returns a full tournament object
+		const players = await ChallongeTournament.findOneOrFail(tournamentId, { select: ["confirmed"] });
+		return players.confirmed;
+	}
+
```

Comment:
Since you're looking for `ConfirmedParticipant`, you shouldn't query against the `ChallongeTournament` table but the `ConfirmedParticipant` table.
```ts
await ConfirmedParticipant.find({ tournamentId });
```